CMakeLists.txt

```diff
@@ -45,9 +45,12 @@ target_include_directories(azure_ulib_c
         ${PROJECT_SOURCE_DIR}/pal/${ULIB_PAL_DIRECTORY}
         ${PROJECT_SOURCE_DIR}/pal/os/inc
         ${PROJECT_SOURCE_DIR}/pal/os/inc/${ULIB_PAL_OS_DIRECTORY}
-    PRIVATE
-        ${PROJECT_SOURCE_DIR}/deps/umock-c/inc
-        ${PROJECT_SOURCE_DIR}/deps/azure-macro-utils-c/inc
+)
+
```

Comment:
Should this CMakeLists have find_package for umock-c and macro-utils then?

Reply:
darn you're probably right. i'll add the installation case (`find_package`)